packages/vc-test-suite/httpService.ts

```diff
@@ -1,12 +1,22 @@
 import axios, { AxiosRequestConfig } from 'axios';
+import _ from 'lodash';
```

Comment:
Hi, maybe we forgot to install this library?
<img width="976" alt="image" src="https://github.com/user-attachments/assets/c8cf0e81-cdcd-47f4-adce-e43873567246">

<img width="951" alt="image" src="https://github.com/user-attachments/assets/ad957bba-c04f-4c72-bf0b-9d3833558505">